OpenTelemetry.AutoInstrumentation.nuspec

```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>OpenTelemetry.AutoInstrumentation</id>
+    <version>0.3.1-beta.1</version>
```

Comment:
Extend releasing.md to include this version.

Reply:
We really should port auto update target 😄